forge-game/src/main/java/forge/game/ability/effects/RepeatEachEffect.java

```diff
@@ -100,7 +100,9 @@ else if (sa.hasParam("DefinedCards")) {
 
         if (loopOverCards) {
             if (sa.hasParam("ChooseOrder") && repeatCards.size() >= 2) {
-                repeatCards = player.getController().orderMoveToZoneList(repeatCards, ZoneType.Stack, sa);
+                final Player chooser = sa.getParam("ChooseOrder").equals("True") ? player :
```

Comment:
@Northmoc 
I hate being this pendantic but to fully follow the ruling the order should still be changeable after each Dig
Though in 99% of games that will hardly matter, so I'll leave that decision at your discretion :P